src/data_conn.c

```diff
@@ -267,6 +270,9 @@ uzfs_submit_writes(zvol_info_t *zinfo, zvol_io_cmd_t *zio_cmd)
 
 		/* IO to clone should be sent only when it is from app */
 		if (!is_rebuild && !ZVOL_IS_HEALTHY(zinfo->main_zv)) {
+			if (IS_ZVOL_OFFLINE(zinfo->clone_zv))
```

Comment:
lets remove this to make sure writes received from app completes fully but not partially.

Reply:
done